tests/test_project.py

```diff
@@ -56,6 +56,27 @@ def suspend_logging():
         logging.disable(logging.NOTSET)
 
 
+@contextmanager
+def setup_project_subprocess_execution(
+    project, capture_stderr=True, stderr_output=None
```

Comment:
An alternative would be to let the default behavior be something like `stderr_output=sys.stderr` and eliminate the `capture_stderr` flag?

Also "capture" seems like the wrong word if `stderr_output is None`, because the "captured" result is just discarded. Hmm.

Maybe the default can be `None` or `sys.stderr`, and if we don't want the output to be shown or captured at all, we do something like redirect to `os.devnull`?

Reply:
I dislike the amount of `stderrr` capturing we do as it makes debugging tests difficult, so I would be happy to change the default behavior.